setup.py

```diff
@@ -5,9 +5,9 @@
 install_requires = [
     'lxml',
     'setuptools',
-    'Django>=1.4',
+    'Django>=1.8',
```

Comment:
I'd rather continue to support < 1.8 on the new patch release and then officially drop < 1.8 on a major release